content/md/what-is/what-about-products-variants.md

```diff
@@ -24,52 +24,51 @@ There are more and more products with variants in all businesses, here are some
 - Teas by packaging
 - Fruit compotes with different flavors
 
-# How we model products with variants?
+# How do we model products with variants?
 
-Let's take a well-known example of products with variants that are t-shirts. A t-shirt model is available in 3 colors (grey, blue, red) and different sizes (S, M, L).
+Let's take a well-known example of products with variants that are T-shirts. A T-shirt model is available in 3 colors (grey, blue, red) and different sizes (S, M, L).
 
-For all the colors and sizes of the t-shirt, some attributes are common, such as the name « Cotton t-shirt with a round neck », the brand « DIVIDED », the care instructions « machine wash at fourty degrees ».   
-Each t-shirt color has different pictures and could have a different composition.  
-Each t-shirt color could be available in different sizes: S, M or L.   
-For each color/size, the identifier of the product is different (EAN, SKU) and the technical dimensions like weight, sleeves length could be different.  
+For all the colors and sizes of the T-shirt, **some attributes are common**, such as the name « Cotton T-shirt with a round neck », the brand « DIVIDED », and the care instructions « machine wash at fourty degrees ».   
+Each T-shirt color has **different pictures and could have a different composition**.  
+Each T-shirt color could be **available in different sizes: S, M or L.**   
+For each color/size, **the identifier of the product is different** (EAN, SKU) as well as **the technical specs like weight, sleeves length that could vary from a T-shirt from another.**
 
 ![Scheme modeling products with variants](../img/scheme_variants.png)
 
-## What is a product model?
+## What is a Product Model?
 
-A **product model** gathers similar products, that differ in some aspects, and allows the **enrichment of their common properties**.
-A product model has no product identifier (ex: SKU) but a dedicated code.
+A **Product Model** gathers similar products, that differ in some aspects (size, color, price...). A Product Model is used to **enrich these products' common properties**.
+A product model has **no product identifier** (ex: SKU) but a dedicated code.
 
-## What is a variant product?
+## What is a Variant Product?
 
-A **variant product** is a product, it's a **variant of a product model** and it has all the product properties (common and variant).    
-A variant product has a product identifier (ex: SKU).
+A **Variant Product** is a product attached to a Product Model. It shares the common attributes of the Product Model and has also its own properties. A **variant product has a product identifier** (ex: SKU).
```

Comment:
Why the use of uppercase here?

Reply:
To highlight them :) I thought it could be a good idea, if you don't like it I'll put back a lower case :)

Reply:
I think it's better to keep lower case, an easier rule to write the articles.

Reply:
WIP :)